### PR TITLE
glfs: update the glfs logdir

### PR DESCRIPTION
**Pull request by lxbsz**
Update README to add uninstall info.
Update the logdir in glfs handler when the logdir is changed.

### CODE REVIEW

**Comment by lxbsz** [comment]
@mikechristie @amarts @pkalever @nixpanic
Please review.
Thanks.

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ We encourage pull requests and issues tracking via Github, and the [target-devel
    * *Note:* If using systemd, `-DSUPPORT_SYSTEMD=ON -DCMAKE_INSTALL_PREFIX=/usr` should be passed to cmake, so files are installed to the correct location.
 1. Type `make`
 1. Type `make install`
+1. Type `xargs rm < install_manifest.txt` to uninstall from source
```
Comment (nixpanic) [inline README.md]: Is `make uninstall` not functional?

File: glfs.c [hunk 2]
```diff
@@ -196,6 +194,33 @@ static int gluster_cache_add(gluster_server *dst, glfs_t *fs, char* cfgstring)
 	return -1;
 }
 
+static bool gluster_cache_update_logdir(const char *logdir)
```
Comment (nixpanic) [inline glfs.c]: Should `logdir` not be used somewhere in this function? Now it is only printed when there is an error.

File: README.md (hunk 1 above)
Comment (lxbsz) [reply README.md]: @nixpanic
Yeah, for the cmake this is not functional as default, please see [1].

[1] https://gitlab.kitware.com/cmake/community/wikis/FAQ#can-i-do-make-uninstall-with-cmake

Or just add one as the wiki does ? Not sure whether will it make sense here by make it more complicated.

Thanks.

File: glfs.c (hunk 2 above)
Comment (lxbsz) [reply glfs.c]: Yeah, right and will fix it.
This is from the old version and forget to remove it.
Thanks.

**Comment by nixpanic** [comment]
> +1. Type `xargs rm < install_manifest.txt` to uninstall from source
> 
> @nixpanic
> Yeah, for the cmake this is not functional as default, please see [1].

In that case, I think the `xargs rm < install_manifest.txt` is fine.

**Comment by lxbsz** [comment]
> +1. Type `xargs rm < install_manifest.txt` to uninstall from source @nixpanic Yeah, for the cmake this is not functional as default, please see [1].
> In that case, I think the `xargs rm < install_manifest.txt` is fine.

Cool, Thanks.
BRs

File: libtcmu_log.h [hunk 3]
```diff
@@ -14,6 +14,8 @@
 #include <stdarg.h>
 #include <stdbool.h>
 
+#include "libtcmu_config.h"
```
Comment (mikechristie) [inline libtcmu_log.h]: Just do a forward declartion for struct tcmu_config like we do for struct tcmu_device.

Comment (lxbsz) [reply libtcmu_log.h]: Sure, will fix it.
Thanks.

**Pull request merged**